components/Header/Header.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import hamburgerMenu from "@/public/hamburgerMenu.svg";
+import Image from "next/image";
+import Navigation from "../Navigation/Navigation";
+import { Container, StyledMenuIcon } from "./Header.styled";
+
+export default function Header() {
+  const [sidebar, setSidebar] = useState(false);
+
+  const showSidebar = () => setSidebar(!sidebar);
+
+  return (
+    <Container>
+      <StyledMenuIcon>
+        <Image
+          src={hamburgerMenu}
+          width={30}
+          height={30}
+          alt="Hamburger Menu"
+          onClick={showSidebar}
+        />
+        {sidebar && <Navigation active={setSidebar} />}
```

Comment:
The `active` prop could be named more appropriately. I understand that the Navigation can be `active` in that it's open or closed. But since this in an **action** you want to call the prop something with _**on**_, like `onClose`.

You could even implement the behavior right here if you want.
```suggestion
        {sidebar && <Navigation onClose={() => setSidebar(false)} />}
```
That way your `Navigation` component doesn't have to implement the logic itself. It receives the expected behavior and merely has to use it.